components/job-orchestration/job_orchestration/scheduler/job_config.py

```diff
@@ -35,4 +36,5 @@ class SearchConfig(BaseModel):
     begin_timestamp: typing.Optional[int] = None
     end_timestamp: typing.Optional[int] = None
     ignore_case: bool = False
+    tags: typing.Optional[typing.List[str]] = None
```

Comment:
Minor, but we can move this above `begin_timestamp` for consistency with all the other places.